src/next-child-view-container.js

```diff
@@ -0,0 +1,129 @@
+import _ from 'underscore';
+import emulateCollection from './utils/emulate-collection';
+
+// Provide a container to store, retrieve and
+// shut down child views.
+const Container = function() {
+  this._views = [];
+  this._viewsByCid = {};
+  this._indexByModel = {};
+  this._updateLength();
+};
+
+emulateCollection(Container.prototype, '_views');
+
+function stringComparator(comparator, view) {
+  return view.model && view.model.get(comparator);
+}
+
+// Container Methods
+// -----------------
+
+_.extend(Container.prototype, {
+
+  // Add a view to this container. Stores the view
+  // by `cid` and makes it searchable by the model
+  // cid (and model itself). Additionally it stores
+  // the view by index in the _views array
+  _add(view, index) {
+    const viewCid = view.cid;
+
+    // store the view
+    this._viewsByCid[viewCid] = view;
+
+    // index it by model
+    if (view.model) {
+      this._indexByModel[view.model.cid] = viewCid;
+    }
+
+    // add to end by default
+    index = index || this._views.length;
+    this._views.splice(index, 0, view);
+
+    this._updateLength();
+  },
+
+  // Sort (mutate) and return the array of the child views.
+  _sort(comparator) {
+    if (typeof comparator === 'string') {
+      comparator = _.partial(stringComparator, comparator);
```

Comment:
hmm yeah I dunno.  `_.partition` isn't core either.  I don't know how much I care about core since it didn't affect underscore.  Do you have a preference?  Would we need to list what lodash modules outside of core we're using?